Could not find module ‘Network’

```
src/Network/Bitcoin/BitX/Internal.hs:26:1: error:
    Could not find module ‘Network’
    Use -v to see a list of the files searched for.
   |
26 | import Network (withSocketsDo)
```

--- a/src/Network/Bitcoin/BitX/Internal.hs
+++ b/src/Network/Bitcoin/BitX/Internal.hs
@@ -23,7 +23,7 @@ import qualified Data.ByteString.Lazy as BL
 import Data.ByteString.Lazy (toStrict)
 import qualified Data.ByteString as BS
 import Data.Maybe (fromJust)
-import Network (withSocketsDo)
+import Network.Socket (withSocketsDo)
 import qualified Data.Text.Encoding as Txt
 import qualified Data.Text as Txt
 import Network.Bitcoin.BitX.Response

